tests/smoke/accessibility.tests.js

```diff
@@ -0,0 +1,249 @@
+        await expect(test.settingsScreen.ConfigurationHeader()).toExist();
+    });
+    it('has a pump header', async () => {
+        await expect(test.settingsScreen.PumpHeader()).toExist();
+    });
+    it('has a pump header', async () => {
+        await expect(test.settingsScreen.ContinuousGlucoseMonitorHeader()).toExist();
+    });
+    it('has a settings header', async () => {
+        await expect(test.settingsScreen.SettingsHeader()).toExist();
+    });
+    it('has a Issue Report Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.IssueReportLabel()).toExist();
+    });
+    it('has a Correction Range Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.CorrectionRangeLabel()).toExist();
+    });
+    it('has a Suspend Threshold label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.SuspendThresholdLabel()).toExist();
+    });
+    it('has a Basal Rates Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.BasalRatesLabel()).toExist();
+    });
+    it('has a Delivery Limits Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.DeliveryLimitsLabel()).toExist();
+    });
+    it('has a Insulin Model Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.InsulinModelLabel()).toExist();
+    });
+    it('has a Carb Ratios Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.CarbRatiosLabel()).toExist();
+    });
+    it('has a Insulin Sensitivities Label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.InsulinSensitivitiesLabel()).toExist();
+    });
+    it('has a Closed Loop button', async () => {
+        await expect(test.settingsScreen.ClosedLoopButton()).toExist();
+    });
+    it('has a Issue Report button', async () => {
+        await expect(test.settingsScreen.IssueReportLabel()).toExist();
+    });
+    it('has a Add Pump label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.AddPumpLabel()).toExist();
+    });
+    it('has a Add CGM label', async () => {
+        //TODO: this is acting like a button!
+        await expect(test.settingsScreen.AddCGMLabel()).toExist();
+    });
+    it('has a services header', async () => {
+        await test.settingsScreen.ScrollToBottom();
+        await expect(test.settingsScreen.ServicesHeader()).toExist();
+    });
+    it('has to close the settings screen', async () => {
+        await test.settingsScreen.Close();
+    });
+};
+
+var settingsCarbRatiosScreenAccessibilityTests = (test) => {
+    var screen;
+    it('open', async () => {
+        await test.settingsScreen.Open();
```

Comment:
[nit] Consider even having the settings screen be a separate object. So,
```
settingsScreen = await test.homeScreen.OpenSettingsScreen();
screen = await settingsScreen.OpenCarbRatiosScreen();
```
(Note: `OpenCarbRatiosScreen` -> `OpenCarbRatiosScheduleScreen`?)

Reply:
yeah it has worked well so I will do the same for the other screens